src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,96 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int size;
+    private int capacity;
+    private Node<K, V>[] table;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_INITIAL_CAPACITY];
+        capacity = DEFAULT_INITIAL_CAPACITY;
+    }
 
     @Override
     public void put(K key, V value) {
+        if (size >= (table.length * DEFAULT_LOAD_FACTOR)) {
+            resize();
+        }
+        int hash = hash(key);
+        Node<K, V> nodeToPut = new Node<K, V>(hash, key, value, null);
+        Node<K, V> node = table[hash];
+        if (node == null) {
+            table[hash] = nodeToPut;
+            ++size;
+            return;
```

Comment:
you can add `else` and take `++size` out of the if-else block. Also, you rid of `return`.